projects/packages/analyzer/scripts/jp-analyze-parallel.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+ini_set( 'memory_limit', '512M' );
+
+require dirname( __DIR__ ) . '/vendor/autoload.php';
+
+use Automattic\Jetpack\Analyzer\Locker;
+use Automattic\Jetpack\Analyzer\Model;
+use Automattic\Jetpack\Analyzer\PluginDownloader;
+use Automattic\Jetpack\Analyzer\Scripts;
+
+$slurper_path = dirname( __DIR__ ) . '/slurper/plugins';
+
+$jetpack_exclude = array( '.git', 'vendor', 'tests', 'docker', 'bin', 'scss', 'images', 'docs', 'languages', 'node_modules' );
+
+class ScanManager {
+	private $count          = 0;
+	private $procs          = array();
+	private $model;
+
+	public function __construct() {
+		$this->model = new Model();
+	}
+
+	public function start_proc( $folder_name ) {
+		echo 'Starting ' . basename( $folder_name ) . "\n";
+		$cmd = 'php ' . escapeshellarg( dirname( __DIR__ ) . '/scripts/jp-warnings-job.php' ) . ' ' . escapeshellarg( $folder_name );
```

Comment:
Hmm. If we're actually requiring PHP 8 in analyzer now, we could make this
```suggestion
		$cmd = array( 'php', dirname( __DIR__ ) . '/scripts/jp-warnings-job.php', $folder_name );
```
to save some escaping. As of PHP 7.4, `proc_open()` accepts an array.